src/main/java/frc/robot/RobotContainer.java

```diff
@@ -52,27 +50,36 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
   private void configureButtonBindings() {
+    //: Conveyor Control
+    JoystickButton conveyorUp = new JoystickButton(m_controller, XboxController.Button.kLeftBumper.value);
+    conveyorUp.whileHeld(new RunConveyorCommand(m_conveyor, RunConveyorCommand.Direction.kUp));
 
+    JoystickButton conveyorDown = new JoystickButton(m_controller, XboxController.Button.kRightBumper.value);
+    conveyorDown.whileHeld(new RunConveyorCommand(m_conveyor, RunConveyorCommand.Direction.kDown));
+    //: Shooter control
     JoystickButton rpmButton1 = new JoystickButton(m_controller, XboxController.Button.kA.value);
-    rpmButton1.whileHeld(new RunShooterCommand(m_shooter, 3000));
+    rpmButton1.whileHeld(new RunShooterCommand(m_shooter, 1500));
 
     JoystickButton rpmButton2 = new JoystickButton(m_controller, XboxController.Button.kX.value);
-    rpmButton2.whileHeld(new RunShooterCommand(m_shooter, 4000));
+    rpmButton2.whileHeld(new RunShooterCommand(m_shooter, 2000));
 
     JoystickButton rpmButton3 = new JoystickButton(m_controller, XboxController.Button.kB.value);
-    rpmButton3.whileHeld(new RunShooterCommand(m_shooter, 5000));
-
-    /*JoystickButton ForwardClimbClockwiseButton = new JoystickButton(m_controller, XboxController.Button.kY.value);
-    ForwardClimbClockwiseButton.whileHeld(new ForwardClimbClockwise(m_frontClimber));
-
-    JoystickButton ForwardClimbCounterClockwiseButton = new JoystickButton(m_controller, XboxController.Button.kB.value);
-    ForwardClimbCounterClockwiseButton.whileHeld(new ForwardClimbCounterClockwise(m_frontClimber));
-
-    JoystickButton BackwardClimbClockwiseButton = new JoystickButton(m_controller, XboxController.Button.kX.value);
-    BackwardClimbClockwiseButton.whileHeld(new BackwardClimbClockwise(m_backClimber));
-
-    JoystickButton BackwardClimbCounterClockwiseButton = new JoystickButton(m_controller, XboxController.Button.kA.value);
-    BackwardClimbCounterClockwiseButton.whileHeld(new BackwardClimbCounterClockwise(m_backClimber));*/
+    rpmButton3.whileHeld(new RunShooterCommand(m_shooter, 2500));
+    //: Climber control
+    DPadTrigger ForwardClimbClockwiseButton = new DPadTrigger(m_controller, DPadTrigger.DPad.kUp);
+    ForwardClimbClockwiseButton.whileActiveContinuous(new ClimbClockwise(m_frontClimber));
```

Comment:
What does clockwise/counter clockwise mean? How is it defined? 

Perhaps for the climber we should use terminology like `MoveClimberOutward` or `MoveClimberInward`. The climbers we have are inverted from each other so if we can pick something that is clearer it will help prevent us from causing problems later on if we accidentally have the climber move the wrong way.

Reply:
d9c70d42e73a6290b7082afa2fd2dbf5900a5db6 and 8472ab11be2d67c8ad343014e89dc79b56bf0662